### DISCUSSION TITLE
Phalcon 5.0.0-alpha.2 extension dont loaded in windows 10 [BUG]:

### DISCUSSION THREAD

**Discussion by maurosys** [phalcon/cphalcon#15508] [Q&A]
Questions? Forum: https://phalcon.io/forum or Discord: https://phalcon.io/discord

**Describe the bug**
I'm trying use php 8.0.6 NTS (https://windows.php.net/downloads/releases/php-8.0.6-Win32-vs16-x64.zip) in in windows 10, by internal server, but the extension don't run, i got the error:

`
C:\php>php -ri phalcon

Warning: PHP Startup: Unable to load dynamic library 'phalcon' (tried: C:\php\ext\phalcon (N├úo foi poss├¡vel encontrar o m├│dulo especificado), C:\php\ext\php_phalcon.dll (N├úo foi poss├¡vel encontrar o m├│dulo especificado)) in Unknown on line 0

Parse error: syntax error, unexpected end of file in Command line code on line 1
`

**Details**
 - Phalcon version: 5.0.0-alpha.2
 - PHP Version: (`php -v`) PHP 8.0.6 (cli) (built: May  4 2021 23:31:41) ( NTS Visual C++ 2019 x64 )
 - Operating System: Windows 10 Home Single Language x64 / Version 21H1
 - Server: Embedded php server
 - Other related info (Database, table schema):
![phalcon-start-errorpng](https://user-images.githubusercontent.com/4058208/119849267-ddeb3e00-bee2-11eb-8227-cda1e5e8f963.png)
![phpinfo](https://user-images.githubusercontent.com/4058208/119849270-df1c6b00-bee2-11eb-857b-b56eddba067c.png)

**Comment by Jeckerson** [comment]
Did you added phalcon.dll to the `.ini` file?

**Comment by maurosys** [comment]
Yes Jeckerson, just by this I receive the error **Warning: PHP Startup: Unable to load dynamic library 'phalcon'**

**Comment by Jeckerson** [comment]
Then, it seems that you forgot to paste `php_phalcon.dll` inside `extensions` directory.

**Reply by maurosys** [reply]
> Then, it seems that you forgot to paste `php_phalcon.dll` inside `extensions` directory.

The extension dll is in extensions directory,  but it is'nt loaded, this is my error, php.ini correct setting to use it, in PHP-7.4 i use phalcon-4.1, but in php-8 not load.

![phalcon_extension](https://user-images.githubusercontent.com/4058208/120192860-1b590f80-c1f2-11eb-8f3f-8bed7ca7ceec.png)
